models/bot.go

```diff
@@ -21,6 +21,9 @@ type Bot struct {
 	SlackEventsCallbackPath       string            `mapstructure:"slack_events_callback_path"`
 	SlackInteractionsCallbackPath string            `mapstructure:"slack_interactions_callback_path"`
 	SlackListenerPort             string            `mapstructure:"slack_listener_port"`
+	MatterMostToken               string            `mapstructure:"mattermost_token"`
+	MatterMostServer              string            `mapstructure:"mattermost_server"`
+	MatterMostInsecureProtocol    string            `mapstructure:"mattermost_insecure"`
```

Comment:
why not `bool` here? are you anticipating more than a binary choice? if not, changing to `bool` with make code a little cleaner in some places.

Reply:
I used a string because utils.Substitute  only accepts string values.

If I change it to a bool then this line in `core/configure.go` 

`utils.Substitute(bot.MatterMostInsecureProtocol, emptyMap)`

will cause the build to fail.

I just copied what I saw for the other remotes, but if there is a better way I can definitely change it.

Reply:
thanks for checking! not ideal, but something i will change/improve another time.